core/trino-main/src/main/java/io/trino/server/security/AbstractBearerAuthenticator.java

```diff
@@ -80,6 +67,25 @@ public Identity authenticate(ContainerRequestContext request)
         }
     }
 
+    public String extractToken(ContainerRequestContext request)
```

Comment:
static? Maybe it could be extracted to utility class? It is strange that this method is an API of `AbstractBearerAuthenticator`.